src/System.Runtime.WindowsRuntime.UI.Xaml/pkg/System.Runtime.WindowsRuntime.UI.Xaml.pkgproj

```diff
@@ -35,10 +35,24 @@
     <File Include="$(PlaceHolderFile)">
       <TargetPath>runtimes/win/lib/uap10.0.16299</TargetPath>
     </File>
+
+    <InboxOnTargetFramework Include="net45" />
+
     <InboxOnTargetFramework Include="uap10.0.16299" />
     <InboxOnTargetFramework Include="portable-win8+wpa81" />
     <InboxOnTargetFramework Include="win8" />
     <InboxOnTargetFramework Include="wpa81" />
+
+    <PackageFile Include=".\build\net45\System.Runtime.WindowsRuntime.UI.Xaml.targets">
```

Comment:
I think this deserves a comment explaining the purpose.

Reply:
Added